### PR TITLE
[Remove Stream usages: portal-impl]Lps 176129 stream

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-176129

### CODE REVIEW

**Comment by catarineaguiar** [comment]
no regressions found.

File: portal-impl/src/com/liferay/portal/service/impl/UserGroupLocalServiceImpl.java [hunk 1]
```diff
@@ -1286,31 +1284,32 @@ protected void reindexUserGroup(UserGroup userGroup)
...
+							userIdSet.add(userId);
+						}
+					}
 
-					long[] userIds = longStream.flatMap(
-						userGroupId -> Arrays.stream(
-							getUserPrimaryKeys(userGroupId))
-					).distinct(
-					).toArray();
+					long[] userIds = TransformUtil.transformToLongArray(
```
Comment (dantewang) [inline portal-impl/src/com/liferay/portal/service/impl/UserGroupLocalServiceImpl.java:1311]: @slnn Please try to analyze the business logic, simplify this whole method and reduce iterations.

File: portal-impl/src/com/liferay/portal/upgrade/v7_0_0/UpgradeOrganization.java [hunk 2]
```diff
@@ -119,26 +118,28 @@ protected void upgradeOrganizationSiteHierarchy() throws Exception {
...
+					});
 
 				preparedStatement2.setLong(
 					1,
 					OrganizationGroup.getGroupId(
 						organizationGroup._parentOrganizationId));
-				preparedStatement2.setString(2, groupTreePath);
+				preparedStatement2.setString(
+					2,
+					StringPool.SLASH +
```
Comment (dantewang) [inline portal-impl/src/com/liferay/portal/upgrade/v7_0_0/UpgradeOrganization.java:140]: @slnn Let's just use a StringBundle + for-loop to generate this whole string.

**Comment by dantewang** [comment]
@slnn please resend, thanks.

**Comment by slnn** [comment]
Resend  in https://github.com/liferay-core-infra/liferay-portal/pull/3743

**Pull request closed without merging**